doc/installation.rst

```diff
@@ -5,11 +5,15 @@ Installation Guide
 Spyder is relatively easy to install on Windows, Linux and macOS.
 Just make sure to read and follow these instructions with care.
 
-If you run into problems, before posting a report, *please* consult our comprehensive `Troubleshooting Guide`_ and search the `issue tracker`_ for your error message and problem description.
-These methods generally fix or isolate the great majority of install-related difficulties.
-Thanks!
+.. important::
+
+   Spyder now offers :ref:`standalone_installers_ref` for Windows and macOS, making it easy to get up and running with the IDE without having to download Anaconda or manually install it in your existing environment.
+   While we still support Anaconda, we recommend this install method on those platforms to avoid most problems with package conflicts and other issues.
+
+   If you do run into problems, before posting a report, *please* consult our comprehensive :doc:`troubleshooting guide<first-steps>` and search the `issue tracker`_ for your error message and problem description.
+   These methods generally fix or isolate the great majority of install-related difficulties.
+   Thanks!
```

Comment:
I think this makes paragraph makes the note too long and doesn't really apply to this section. So I suggest to remove it.

Reply:
Yeah, I see that. What about if we just move the second paragraph back to the introduction outside the note, where it was originally?

Reply:
Ok, I agree with that.

Reply:
Done, thanks